pluginconfig/commit.go

```diff
@@ -121,6 +129,12 @@ func (c CommitOffchainConfig) Validate() error {
 			c.TokenPriceBatchWriteFrequency, c.TokenPriceChainSelector)
 	}
 
+	for token := range c.PriceSources {
+		if _, exists := c.TokenDecimals[token]; !exists {
+			return fmt.Errorf("missing TokenDecimals for account: %s", token)
+		}
+	}
+
```

Comment:
One more validation to add here is to validation each `ArbitrumPriceSource` passed in:

```golang
for token, aps := range c.PriceSources {
  if err := aps.Validate(); err != nil {
    return fmt.Errorf("invalid arbitrum price source for token %s: %w", token, err)
  }
}
```